yt_dlp/extractor/tubetugraz.py

```diff
@@ -0,0 +1,349 @@
+# coding: utf-8
+from __future__ import unicode_literals
+from .common import InfoExtractor
+from ..utils import ExtractorError, urlencode_postdata, try_get
+from collections import defaultdict
+import re
+
+
+def try_get_or(src, getters, expected_type=None):
+    for getter in getters:
+        v = try_get(src, getter, expected_type)
+        if v is not None:
+            return v
```

Comment:
try_get already allows multiple getters

fyi: You can use traverse_obj to make it even simpler

Reply:
Thanks, I'll look into it

Reply:
fixed, thank you